packages/core/src/renderer/components/+config-autoscalers/hpa-details.tsx

```diff
@@ -85,11 +85,13 @@ class NonInjectedHpaDetails extends React.Component<HpaDetailsProps & Dependenci
         }
         case HpaMetricType.External:
           return `${metricName} on ${JSON.stringify(metric.external.metricSelector ?? metric.external.metric?.selector)}`;
+        default:
+          return hpa.spec?.targetCPUUtilizationPercentage ? "CPU Utilization percentage" : "unknown";
```

Comment:
Nit: I would check to make sure it is `typeof X === "number"` instead but I doubt that someone would set `targetCPUUtilizationPercentage` as `0`. So probably not important